products/platform/documentation-site/research/search-design-doc.md

```diff
@@ -1,15 +1,114 @@
 # Global Documentation Search
 
-Author: Bill Fienberg
-Last Updated: 2020-01-23
-Reviewers: TBD
+**Author(s):** Bill Fienberg <bill.fienberg@oddball.io>
 
-## Objective
+**Last Updated:** 2020-01-23
 
-## Background
+**Status:** **Draft**
+
+**Approvers:** Andrew G., Megan K., Rian F., DevOps
 
 ## Overview
 
-## Detailed Design
+### Objective
+
+Search multiple documentation sources from a single, public entrypoint.
```

Comment:
What's the objective of this design document? To propose a specific solution (e.g. Algolia) or foster discussion so we can settle on a solution?